src/components/Map/Controls/index.tsx

```diff
@@ -224,4 +253,32 @@ const styles: IStyles = {
   pointerAll: {
     pointerEvents: "all",
   },
+  fixedMidBottom: () => ({
+    position: "fixed",
+    bottom: "32px",
+    left: "0px",
+    width: "100%",
+    zIndex: 1030,
+    display: "flex",
+    flexDirection: "column",
+    justifyContent: "center",
+    alignItems: "center",
+    pointerEvents: "none",
+  }),
+  infoSnackbar: {
```

Comment:
In smaller screens (f.e Galaxy Fold), close button overlaps the content

Reply:
Nice catch 👍
I've fixed it by using flexbox instead of absolute positioning, what do you think?

## Before

https://github.com/acikkaynak/musahit-harita-frontend/assets/1475786/40797f06-828d-48a2-971a-f88e7840eac7

## After

https://github.com/acikkaynak/musahit-harita-frontend/assets/1475786/e325ac97-86b2-415b-ba30-d20076022b33

Reply:
Looks great! Thank you